NangoHQ/nango
# feat(fleet): add /rollout endpoint to server

adding /rollout endpoint 



# How to test
1. make sure you have `NANGO_INTERNAL_API_KEY` env var setup locally
2. run Nango locally
3. `curl -X POST http://localhost:3003/internal/fleet/nango_runners/rollout -H 'Content-Type: application/json' -d '{ "commitHash": "7ed44fc18fc76c66631ac3d4169107b1a48f9c58" }' -H 'Authorization: Bearer XXX'
4. observer new runner to be created and old one to be terminated
